modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -1480,9 +1480,9 @@
                                   "properties": {
                                     "email": {
                                       "type": "string",
-                                      "pattern": "^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$",
+                                      "pattern": "^[^\\s@]+@([^\\s@.,]+\\.)+[^\\s@.,]{2,}$",
```

Comment:
Taken from From 526 

Related slack [discussion](https://oddball.slack.com/archives/C019XKGS1CN/p1733266016398199?thread_ts=1733232536.293779&cid=C019XKGS1CN) for using this